File: Java/백준/Bronze/30793. gahui and sousenkyo 3/gahui and sousenkyo 3.java
import java.util.Scanner;

public class Main{
    public static void main(String[] args){
        Scanner in = new Scanner(System.in);
        int a = in.nextInt();
        int b = in.nextInt();
        if(a<b*0.2){
            System.out.println("weak");
        }
        else if(a<b*0.4){
            System.out.println("normal");
        }
        else if(a<b*0.6){
            System.out.println("strong");
        }
        else{
            System.out.println("very strong");
        }
    }
}

File: C#/백준/Bronze/30793. gahui and sousenkyo 3/gahui and sousenkyo 3.cs
using System;

namespace q{
    class w{
        static void Main(){
            int[] a = Array.ConvertAll(Console.ReadLine().Split(), int.Parse);
            if(a[0]<a[1]*0.2){
                Console.WriteLine("weak");
            }
            else if(a[0]<a[1]*0.4){
                Console.WriteLine("normal");
            }
            else if(a[0]<a[1]*0.6){
                Console.WriteLine("strong");
            }
            else{
                Console.WriteLine("very strong");
            }
        }
    }
}